src/components/App.js

```diff
@@ -41,6 +41,12 @@ fetchTopics() {
     }
   })
   .then((response) => this.props.getTopics(response.data.topics))
+  .then((allTopics) => {
+    return Promise.all([allTopics, this.chooseTopic(allTopics)])
```

Comment:
So, like technically, this isn't wrong, but the first thing (`allTopics` in this case, `chosenTopic` in the next one) has already resolved. `Promise.all` is waiting for _all_ the promises to resolve, but in both these cases, all but one has already resolved.